clt_tests/tests/test-extra.rec

```diff
@@ -0,0 +1,52 @@
+––– block: ./base/init –––
+––– input –––
+docker run --rm -e EXTRA=0 --name manticore -d manticoresoftware/manticore:current
+––– output –––
+#!/[0-9a-z]{40}$/!#
+––– input –––
+timeout 10 docker logs -f manticore 2>1 | grep -qm1 "accepting connections" && echo 'accepting connections' || echo 'Сonnection failed'
```

Comment:
`Connection failed` is confusing since we are not connection anywhere here. The least confusing error message in this case would be `Docker failed to start properly in 10 seconds".